bundles/org.openhab.binding.asuswrt/src/main/java/org/openhab/binding/asuswrt/internal/helpers/AsuswrtUtils.java

```diff
@@ -0,0 +1,464 @@
+        return (boolVal != null ? boolVal ? OnOffType.ON : OnOffType.OFF : OnOffType.OFF);
+    }
+
+    /**
+     * Return OnOffType from bool
+     * 
+     * @param boolVal
+     */
+    public static OnOffType getOnOffType(Integer intVal) {
+        return intVal == 0 ? OnOffType.OFF : OnOffType.ON;
+    }
+
+    /**
+     * Return StringType from String
+     * 
+     * @param strVal
+     */
+    public static StringType getStringType(@Nullable String strVal) {
+        return new StringType(strVal != null ? strVal : "");
+    }
+
+    /**
+     * Return DecimalType from Double
+     * 
+     * @param numVal
+     */
+    public static DecimalType getDecimalType(@Nullable Double numVal) {
+        return new DecimalType((numVal != null ? numVal : 0));
+    }
+
+    /**
+     * Return DecimalType from Integer
+     * 
+     * @param numVal
+     */
+    public static DecimalType getDecimalType(@Nullable Integer numVal) {
+        return new DecimalType((numVal != null ? numVal : 0));
+    }
+
+    /**
+     * Return DecimalType from Long
+     * 
+     * @param numVal
+     */
+    public static DecimalType getDecimalType(@Nullable Long numVal) {
+        return new DecimalType((numVal != null ? numVal : 0));
+    }
+
+    /**
+     * JSON Object to PercentType
+     * 
+     * @param numVal value 0-100
+     * @return PercentType
+     */
+    public static PercentType getPercentType(@Nullable Integer numVal) {
+        Integer val = limitVal(numVal, 0, 100);
+        return new PercentType(val);
+    }
+
+    /**
+     * Return HSBType from integers
+     * 
+     * @param hue integer hue-color
+     * @param saturation integer saturation
+     * @param brightness integer brightness
+     * @return HSBType
+     */
+    public static HSBType getHSBType(Integer hue, Integer saturation, Integer brightness) {
+        DecimalType h = new DecimalType(hue);
+        PercentType s = new PercentType(saturation);
+        PercentType b = new PercentType(brightness);
+        return new HSBType(h, s, b);
+    }
+
+    /**
+     * Return QuantityType with Time
+     * 
+     * @param numVal Number with value
+     * @param unit TimeUnit (Unit<Time>)
+     * @return QuantityType<Time>
+     */
+    public static QuantityType<Time> getTimeType(@Nullable Number numVal, Unit<Time> unit) {
+        return new QuantityType<>((numVal != null ? numVal : 0), unit);
+    }
+
+    /**
+     * Return QuantityType with Power
+     * 
+     * @param numVal Number with value
+     * @param unit PowerUnit (Unit<Power>)
+     * @return QuantityType<Power>
+     */
+    public static QuantityType<Power> getPowerType(@Nullable Number numVal, Unit<Power> unit) {
+        return new QuantityType<>((numVal != null ? numVal : 0), unit);
+    }
+
+    /**
+     * Return QuantityType with Energy
+     * 
+     * @param numVal Number with value
+     * @param unit PowerUnit (Unit<Power>)
+     * @return QuantityType<Energy>
```

Comment:
You should not repeat the parameter/result types in JavaDocs. Describe how parameters are used for creating the result. If that is too much work for simple methods like this just remove the JavaDocs or add one proper sentence the describes what it does.